DEVELOPMENT.md

```diff
@@ -246,7 +246,7 @@ to be edited by hand.
 
 4.  Rebuild and test TensorBoard to make sure it works:
     * `rm -rf node_modules; bazel clean --expunge; yarn`
-    * `bazel run tensorboard --logdir <your favorite logdir>`
+    * `bazel run tensorboard -- --logdir <your favorite logdir>`
```

Comment:
I imagine this is actually a big deal for people not familiar with `bazel` good catch.